File: src/java/eu/sig/training/ch06/simpledigitalcamera/SmartphoneApp.java
package eu.sig.training.ch06.simpledigitalcamera;

import java.awt.Image;

@SuppressWarnings("unused")
// tag::SmartphoneApp[]
public class SmartphoneApp {
    private static SimpleDigitalCamera camera = SDK.getCamera();

    public static void main(String[] args) {
        // ...
        Image image = camera.takeSnapshot();
        // ...
    }
}
// end::SmartphoneApp[]

File: src/csharp/eu/sig/training/ch06/simpledigitalcamera/SmartphoneApp.cs
using System.Drawing;

namespace eu.sig.training.ch06.simpledigitalcamera
{

#pragma warning disable 219
    // tag::SmartphoneApp[]
    public class SmartphoneApp
    {
        private static ISimpleDigitalCamera camera = SDK.GetCamera();

        public static void Main(string[] args)
        {
            // ...
            Image image = camera.TakeSnapshot();
            // ...
        }
    }
    // end::SmartphoneApp[]
#pragma warning restore 219
}
